app/jobs/inboxes/fetch_imap_emails_job.rb

```diff
@@ -26,7 +26,7 @@ def process_mail_for_channel(channel)
     Mail.defaults do
```

Comment:
can we also take a quick look at the #TODO comment here ? possibly something like

```
mailer_retriever = Mail.retriever_method.new :imap, address: channel.imap_address,
                              port: channel.imap_port,
                              user_name: channel.imap_email,
                              user_name: channel.imap_login,
                              password: channel.imap_password,
                              enable_ssl: channel.imap_enable_ssl
mail_retriever.find ....
```

Reply:
@sojan-official can we take this as a separate issue?